ts/test/scenario.ts

```diff
@@ -346,6 +346,83 @@ test("pauseUntilDhtEqual - Create multiple entries, read the last, 2 conductors"
   await scenario.cleanUp();
 });
 
+test("dhtSync - Fails if some Ops are not synced among all conductors", async () => {
+  const scenario = new Scenario({
+    disableLocalServices: true,
+  });
+
+  const appBundleSource: AppBundleSource = {
+    type: "path",
+    value: FIXTURE_HAPP_URL.pathname,
+  };
+  const [alice, bob] = await scenario.addPlayersWithApps([
+    { appBundleSource },
+    { appBundleSource },
+  ]);
+
+  // Alice creates 1 entry
+  await alice.cells[0].callZome<EntryHash>({
+    zome_name: TEST_ZOME_NAME,
+    fn_name: "create",
+    payload: "my entry",
+  });
+
+  // Bob never receives the entry
+  try {
+    await dhtSync([alice, bob], alice.cells[0].cell_id[0]);
+    assert.fail();
+  } catch (e) {
+    assert(true);
+  }
+
+  await scenario.cleanUp();
+});
+
+test("dhtSync - Fails if some Ops are not integrated in a conductor", async () => {
```

Comment:
I don't think is actually testing what I want, since Alice has these ops integrated but Bob doesn't. It would have failed before these changes.